prediction_market_agent/agents/microchain_agent/microchain_agent.py

```diff
@@ -261,7 +270,7 @@ def step_end_callback(agent: Agent, step_output: StepOutput) -> None:
         engine_help=agent.engine.help
     )
     if bootstrap:
-        agent.bootstrap = [bootstrap]
+        agent.bootstrap.append(bootstrap)
```

Comment:
Here as well, bootstrap is `list` in Microchain and previously it would override it.